plugins/smart_proxy_ansible/control

```diff
@@ -11,7 +11,8 @@ Package: ruby-smart-proxy-ansible
 Architecture: all
 XB-Ruby-Versions: ${ruby:Versions}
 Depends: ${shlibs:Depends}, ${misc:Depends}, ruby | ruby-interpreter, foreman-proxy (>= 1.22.0~rc1),
-  ruby-smart-proxy-dynflow (>= 0.1.0), ruby-smart-proxy-dynflow (<< 1.0.0),
-  ruby-foreman-ansible-core,
+  ruby-smart-proxy-dynflow (>= 0.5.0), ruby-smart-proxy-dynflow (<< 1.0.0),
+  ruby-foreman-ansible-core, ruby-net-ssh,
```

Comment:
do we want to keep `foreman-ansible-core`?

Reply:
my understanding is, this should go away

Reply:
It should go away, I kept it around as I wasn't sure how to proceed with this